typings/index.d.ts

```diff
@@ -277,6 +277,7 @@ export class Command {
   processedArgs: any[];
   commands: Command[];
   parent: Command | null;
+  readonly options: Option[];
```

Comment:
If you still want to add options, I would like it after commands (one line higher) and without the readonly.